foxpuppet/browser/navbar.py

```diff
@@ -0,0 +1,41 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+
+MENU_BUTTON = 'PanelUI-menu-button'
+
+
+class Navbar(object):
+
+    def __init__(self, selenium, *args, **kwargs):
+        self.selenium = selenium
+        self._locationbar = None
+        self.wait = WebDriverWait(selenium, 10)
```

Comment:
We haven't used the menu button in our tests yet but I can assume it happens because buttons are getting populated slower than usual on some Travis nodes.